packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -1613,3 +1627,15 @@ cp1Rusd_sdeusdParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_sdeusdParentConfig_priceHaircutUnscaled = "0.075"
 cp1Rusd_sdeusdParentConfig_autoExchangeDiscountUnscaled = "0.005"
 cp1Rusd_sdeusdParentConfig_oracleNodeId = "<%= settings.sdeusdUsdcNodeIdStork %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_rselini_capUnscaled = "<%= formatEther(MaxUint256) %>"
```

Comment:
hmm, to be safe i think we should just set the cap initially to 2m since we intend to send 2m of selini tokens via the passive pool into the core

Reply:
same's true for amber, wdyt?

Reply:
this is reya cronos

Reply:
i know but let's make sure it's mirror of mainnet as much as possible (if we go with 2m for mainnet, then need to make sure it's also 2m here)